samples/kubernetes/end2end/demo-domains/domainBuilder/Dockerfile

```diff
@@ -0,0 +1,27 @@
+
+FROM store/oracle/weblogic:12.2.1.3
```

Comment:
FROM container-registry.oracle.com/middleware/weblogic:12.2.1.3

Reply:
In the guide I added step to tag he image as store/oracle/weblogic:12.2.1.3.

Reply:
No we should be pulling the image from OCR instead of from the store.  Please look at new documentation in Operator 2.3.

Reply:
See https://github.com/oracle/weblogic-monitoring-exporter/blob/revert-31-revert-30-end2end-sample/samples/kubernetes/end2end/docs/03-wls-operator.md#installing-the-wls-operator. Yes, the image is pulled from OCR but the image is tagged to store/oracle/weblogic:12.2.1.3.

Reply:
Fixed.